arangod/Agency/AgencyFeature.cpp

```diff
@@ -168,6 +170,28 @@ regular cluster, which needs to handle only a part of the overall load.)");
                          arangodb::options::Flags::DefaultNoComponents,
                          arangodb::options::Flags::OnAgent));
 
+  options
+      ->addOption(
+          "--agency.supervision-delay-add-follower",
+          "delay in supervision, before an AddFollower job is executed [s]",
```

Comment:
@Simran-B recently merged a PR that unifies the casing of all startup options descriptions. Convention is now to start with a capital letter and end the description with a `.`
```suggestion
          "The delay in supervision, before an AddFollower job is executed (in seconds).",
```